util/PostgresMigrations/HelperScripts/2024-08-12_00_FinalFlexibleCollectionsDataMigrations.psql

```diff
@@ -0,0 +1,38 @@
+﻿-- Migrate Custom users who only have 'editAssignedCollections' and/or 'deleteAssignedCollections' custom permissions to the User type.
+UPDATE "OrganizationUser"
+SET
+    "Type" = 2,
+    "Permissions" = NULL
+WHERE
+    "Type" = 4
+    AND jsonb_typeof("Permissions"::jsonb) = 'object'
+    AND (
+        COALESCE(("Permissions"::jsonb)->>'editAssignedCollections', 'false') = 'true'
+        OR COALESCE(("Permissions"::jsonb)->>'deleteAssignedCollections', 'false') = 'true'
```

Comment:
I assume `COALESCE(..., 'false')` is required in case the value is missing/null?

Reply:
Yes

Reply:
Actually, I've just tested the script without `COALESCE` and it still works.